src/pages/accounting/BalancesTable.tsx

```diff
@@ -34,10 +35,11 @@ function BalancesTable({ accounting, isLoadingUncashed }: Props): ReactElement |
             <TableCell style={{ textAlign: 'right' }}>Settlements Sent / Received</TableCell>
             <TableCell style={{ textAlign: 'right' }}>Total</TableCell>
             <TableCell style={{ textAlign: 'right' }}>Uncashed Amount</TableCell>
+            <TableCell style={{ textAlign: 'right' }}></TableCell>
           </TableRow>
         </TableHead>
         <TableBody>
-          {Object.entries(accounting).map(([peer, values]) => (
+          {accounting.map(({ peer, balance, received, sent, uncashedAmount }) => (
```

Comment:
```suggestion
          {accounting?.map(({ peer, balance, received, sent, uncashedAmount }) => (
```

Reply:
It can not be null here, it is already checked for null previously.